allensdk/brain_observatory/gaze_mapping/_gaze_mapper.py

```diff
@@ -191,9 +191,6 @@ def pupil_pos_in_eye_coords(self,
         delta_py = pupil_cr_delta.T[1]
 
         R_eye_to_cam = self.camera.generate_self_to_eye_frame_xform().inv()
-        # Camera captures images of eye off of a dichroic mirror
-        # so image will appear rotated 180 degrees about its y-axis.
-        self.camera.rotations = self.camera.rotations + [0, np.pi, 0]
```

Comment:
In the commit message you talk about 90 degree rotation, but here you removing 180 degree rotation.

Reply:
Really good catch! The commit message and readme were wrong, it should be 180 degrees.